examples/morpheusvm/tests/workload/genesis.go

```diff
@@ -68,15 +68,23 @@ func newDefaultKeys() []ed25519.PrivateKey {
 		}
 		testKeys[i] = ed25519.PrivateKey(bytes)
 	}
-
 	return testKeys
 }
 
+// NetworkConfiguration implements workload.TestNetworkConfiguration interface.
 type NetworkConfiguration struct {
 	workload.DefaultTestNetworkConfiguration
 	keys []ed25519.PrivateKey
 }
 
+func (n *NetworkConfiguration) PrivateKeys() []*auth.PrivateKey {
+	keys := make([]*auth.PrivateKey, 0, len(n.keys))
+	for _, key := range n.keys {
+		keys = append(keys, auth.NewPrivateKeyFromED25519(key))
+	}
+	return keys
+}
+
```

Comment:
Would it make sense to move the added keys into `workload.DefaultTestNetworkConfiguration` rather than keeping the wrapping `NetworkConfiguration` type?

Reply:
It depends, the entiere tests of MorpheusVM are manipulating `ed25519` private key right now. Moving it into `workload.DefaultTestNetworkConfiguration` would require to either always store keys as `ed25519` or as generic type `auth.PrivateKey`.
At the first glance, I see no problem in either of the two scenarios.

Reply:
Yes, I think we can use `auth.PrivateKey` or as mentioned elsewhere `AuthFactory` since it includes `Address()` in the interface now.